storage/map_files_downloader.cpp

```diff
@@ -149,10 +149,15 @@ MapFilesDownloader::ServersList MapFilesDownloader::LoadServersList()
 {
   auto constexpr kTimeoutInSeconds = 10.0;
 
-  platform::HttpClient request(GetPlatform().MetaServerUrl());
+  std::string metaServerUrl = GetPlatform().MetaServerUrl();
```

Comment:
nit: In our code style, all constants should be marked as `std:string const`. It helps to read the code and avoid assuming that later this variable is modified somewhere else.

Reply:
Sure, fixed!